src/dumpsxiso/cue.cpp

```diff
@@ -0,0 +1,146 @@
+#include "cue.h"
+#include "platform.h"
+#include <fstream>
+
+bool multiBinSeeker(const unsigned int sector, const cd::IsoDirEntries::Entry &entry, cd::IsoReader &reader, const CueFile &cueFile)
+{
+	unsigned trackIndex = (entry.trackid.empty() ? std::stoi(entry.identifier.substr(6, 2)) : std::stoi(entry.trackid)) - 1;
```

Comment:
The index is unsigned, but `std::stoi` returns a signed value. Is this intentional?